src/parse.zig

```diff
@@ -171,10 +172,14 @@ pub const Node = struct {
         base: Node = Node{ .tag = Tag.value, .tree = undefined },
         start: ?TokenIndex = null,
         end: ?TokenIndex = null,
+        escape_mode: EscapeMode = .None,
```

Comment:
Oh, I just realized this might not be necessary anymore. I should probably remove `escape_mode`